docker-compose.yml

```diff
@@ -152,6 +153,30 @@ services:
       args:
         BASE_IMAGE: 'sentry-onpremise-local'
     command: '"0 0 * * * gosu sentry sentry cleanup --days $SENTRY_EVENT_RETENTION_DAYS"'
+  reverse_proxy:
+    ports:
+      - '9000:80/tcp'
+    image: "nginx:1.16.1"
+    volumes:
+      - type: bind
+        source: ./reverse_proxy_config/nginx.conf
+        target: /etc/nginx/nginx.conf
+    depends_on:
+      - web
+      - relay
+  relay:
+    image: "us.gcr.io/sentryio/relay:latest"
+    ports:
+      - '3000:3000/tcp'
```

Comment:
I don't think we need to expose any ports from `relay` to outside world as it goes through nginx?
```suggestion
```

Reply:
you are right